doc/nrf/gs_modifying.rst

```diff
@@ -181,12 +282,32 @@ Configuring build types
 
 .. build_types_overview_start
 
-Build types enable you to use different sets of configuration options for each board.
-You can create several build type :file:`.conf` files per board and select one of them when building the application.
-This means that you do not have to use one :file:`prj.conf` file for your project and modify it each time to fit your needs.
+The application is defined by a set of configuration files that provide definitions related to software and hardware, which together form the *configuration set* of the application.
+The configuration set specifies which features the application is going to use and how it is going to behave.
+For convenience, you can specify multiple configuration sets to quickly switch the application behavior.
+The most common example is to switch between ``release`` and ``debug`` variants of the application.
+
+In the |NCS|, you can define multiple build types.
```

Comment:
Is there a one-to-one mapping between configuration sets and build types?

Do we need both terms?

Reply:
Unified to *build type*